server/protocols/jmap/src/main/java/org/apache/james/jmap/mailet/VacationReply.java

```diff
@@ -76,9 +77,7 @@ public VacationReply build() throws MessagingException {
 
         private MimeMessage generateMimeMessage() throws MessagingException {
             MimeMessage reply = (MimeMessage) originalMail.getMessage().reply(NOT_REPLY_TO_ALL);
-            if (subject.isPresent()) {
-                reply.setHeader("subject", subject.get());
-            }
+            subject.ifPresent((ThrowingConsumer<String>) subjectString -> reply.setHeader("subject", subjectString));
```

Comment:
In fact it should be Throwing.consumer(subjectString -> reply.setHeader("subject", subjectString)) if I remember correctly.